lib/assets/javascripts/turbograft/remote.coffee

```diff
@@ -73,6 +70,20 @@ class TurboGraft.Remote
 
   uriEncodeForm: (form) ->
     formData = ""
+    @_iterateOverFormInputs form, (input) =>
+      formData = @formAppend(formData, input.name, input.value)
+    formData
+
+  nativeEncodeForm: (form) ->
+    formData = new FormData
+    @_iterateOverFormInputs form, (input) =>
+      if input.type == 'file'
+        formData.append(input.name, input.files[0]) # <- just a guess
```

Comment:
I think we can ship this atm with just the first file, and make an open issue to solve it